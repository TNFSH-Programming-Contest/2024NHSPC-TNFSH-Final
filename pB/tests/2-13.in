89 10
8 10 7 5 4 10 7 9 8 2 6 1 7 8 1 10 10 2 10 4 1 8 6 10 3 2 2 2 1 9 5 2 7 7 4 9 9 4 8 10 9 10 4 2 10 7 10 2 6 3 10 6 4 10 8 3 2 5 1 6 9 4 9 2 7 10 8 6 6 9 9 2 8 10 9 5 8 7 9 10 7 5 6 2 6 6 7 7 2
108392278 1706 129595680 42 4919 94 19 977928062 1838919 109
0 
5 4 10 8 6 3
49
0 
0 
37
0 
2 9 2
4
0 
8 1 5 9 3 10 6 2 4
88
0 
10 9 3 4 6 10 7 5 2 8 1
30
0 
1 4
21
0 
8 2 8 5 3 9 6 4 10
98
0 
7 4 6 7 1 2 3 9
62
0 
10 6 3 8 7 5 1 4 9 10 2
44
0 
10 5 8 1 3 9 4 10 2 6 7
100
0 
6 9 5 1 7 2 6
38
0 
7 7 9 10 2 8 1 5
67
0 
6 2 8 10 6 3 1
44
0 
1 3
23
0 
9 3 6 10 9 4 8 7 1 2
47
0 
9 9 8 2 6 7 1 5 10 3
69
0 
9 9 10 1 3 8 7 2 4 6
66
0 
6 8 7 4 5 3 6
57
0 
9 7 8 1 4 3 10 9 5 6
37
0 
0 
27
0 
3 10 3 9
47
0 
10 8 10 7 4 6 3 9 2 5 1
51
0 
10 5 1 6 7 3 8 4 2 9 10
51
0 
4 10 7 2 9
8
0 
0 
15
0 
1 6
81
0 
6 4 1 3 9 2 7
45
0 
9 7 8 1 5 2 3 10 4 6
79
0 
4 2 10 9 1
85
0 
10 8 7 4 2 3 5 6 10 9 1
72
0 
10 7 2 8 1 10 5 4 6 9 3
72
0 
2 6 4
86
0 
8 1 6 10 3 2 9 8 5
19
0 
2 3 9
28
0 
1 10
97
0 
1 2
56
0 
1 5
91
0 
3 7 10 1
54
0 
0 
40
0 
4 5 1 9 7
55
0 
4 1 9 3 5
33
0 
9 3 10 7 4 5 6 2 1 8
77
0 
5 8 1 9 3 10
41
0 
4 5 7 4 1
68
0 
4 5 6 4 8
81
0 
5 3 8 6 2 10
0
0 
1 5
35
0 
0 
85
0 
4 1 4 5 6
4
0 
6 7 6 5 10 2 8
51
0 
6 3 6 10 5 7 2
41
0 
3 10 3 4
46
0 
9 10 7 5 6 9 1 8 2 4
84
0 
2 4 9
57
0 
5 7 6 4 3 9
44
0 
7 4 1 10 7 8 2 5
87
0 
10 10 4 8 3 2 9 5 7 1 6
68
0 
9 4 6 9 7 10 2 5 1 3
41
0 
8 4 5 9 6 8 7 3 2
55
0 
7 5 6 4 1 8 7 2
42
0 
3 1 10 9
69
0 
0 
8
0 
0 
50
0 
10 4 2 10 6 8 1 5 3 7 9
31
0 
5 9 6 7 1 5
94
0 
3 6 7 9
17
0 
3 7 10 4
34
0 
10 1 4 8 5 2 10 3 7 9 6
85
0 
10 5 1 10 8 7 9 6 3 4 2
92
0 
9 10 8 9 5 3 6 7 2 1
61
0 
5 8 7 3 1 2
85
0 
10 2 6 1 8 5 9 4 10 3 7
82
0 
0 
81
0 
0 
90
0 
6 5 9 1 2 7 6
49
0 
8 5 1 8 9 7 2 6 10
0
0 
10 3 7 1 6 9 10 8 2 4 5
67
0 
8 7 4 10 6 5 3 1 2
87
0 
1 4
17
0 
8 6 10 4 9 1 2 5 7
44
0 
4 5 10 4 9
52
0 
6 10 1 2 6 8 7
1
0 
1 10
27
0 
9 3 2 4 1 9 7 6 5 8
22
0 
9 3 10 5 4 1 8 2 9 6
19
0 
4 10 2 8 4
12
0 
6 4 3 5 1 8 7
26
0 
10 2 10 8 1 7 4 6 3 9 5
2
0 
9 6 1 5 10 8 4 2 7 9
69
0 
0 
50
0 
4 6 9 2 7
36
0 
8 4 5 9 2 1 7 6 10
61
0 
4 7 3 5 4
75
0 
0 
56
0 
2 2 7
94
0 
8 5 6 3 9 2 8 1 10
82
0 
4 3 2 10 1
85
0 
4 7 9 5 1
23
0 
0 
89
0 
8 6 4 9 8 2 10 7 1
78
0 
6 1 5 8 6 10 7
28
0 
9 3 10 7 5 4 6 2 8 9
45
0 
6 4 1 8 2 7 10
72
0 
0 
51
0 
3 8 5 4
71
0 
5 9 2 4 3 10
48
0 
8 9 8 10 6 4 3 7 5
8
0 
7 7 3 2 1 6 4 5
86
0 
7 8 9 2 4 10 5 3
62
0 
3 8 10 1
32
0 
8 9 10 4 5 3 7 6 8
33
0 
4 5 9 7 10
30
0 
5 8 2 10 9 7
25
0 
10 4 1 2 9 6 8 7 5 3 10
15
0 
2 6 8
80
0 
3 2 5 7
66
0 
0 
93
0 
0 
7
0 
9 5 6 7 1 2 4 10 8 3
82
0 
8 10 2 5 1 4 6 8 3
58
0 
6 10 9 2 6 3 1
27
0 
10 7 8 6 1 3 10 2 5 9 4
25
0 
1 1
26
0 
0 
85
0 
6 1 8 9 2 5 10
14
0 
9 7 8 10 4 6 9 3 5 1
35
0 
1 9
23
0 
7 8 1 6 3 2 7 9
84
0 
3 10 1 9
61
0 
4 6 4 3 1
48
0 
6 10 9 4 6 2 5
97
0 
6 2 7 3 5 6 4
1
0 
5 1 10 9 7 5
41
0 
8 7 10 2 3 6 1 9 4
67
0 
10 3 2 6 5 1 4 9 8 7 10
68
0 
9 2 9 7 4 10 8 1 5 6
6
0 
4 7 1 4 8
70
0 
10 3 8 7 1 5 6 10 4 2 9
46
0 
1 2
9
0 
6 10 1 4 7 2 9
95
0 
0 
44
0 
2 7 6
85
0 
5 10 9 6 3 4
85
0 
5 5 8 7 2 6
55
0 
4 5 2 6 8
52
0 
7 5 2 10 4 9 3 6
52
0 
6 5 9 7 8 6 1
8
0 
3 6 3 5
44
0 
9 3 2 4 1 7 9 6 5 10
23
0 
4 5 9 2 3
35
0 
3 1 9 10
19
0 
3 1 3 2
7
0 
0 
92
0 
3 5 1 4
39
0 
2 8 1
13
0 
4 6 8 3 4
6
0 
4 4 2 5 7
57
0 
9 4 1 9 8 7 5 10 3 2
49
0 
6 7 10 6 4 3 9
60
0 
0 
40
0 
8 3 10 8 9 2 7 5 1
14
0 
10 3 6 7 5 1 8 2 9 4 10
14
0 
2 3 7
91
0 
9 2 7 10 3 8 1 6 5 9
28
0 
4 5 8 4 9
31
0 
0 
96
0 
4 1 2 6 10
67
0 
9 3 9 10 6 7 4 8 1 5
7
0 
2 5 9
31
0 
4 8 6 10 2
31
0 
7 1 10 8 7 6 3 9
51
0 
2 3 6
35
0 
8 1 5 7 9 8 3 10 4
19
0 
0 
16
0 
9 6 2 9 10 3 7 5 1 4
74
0 
7 6 3 1 8 2 5 10
49
0 
9 6 2 5 9 4 3 10 8 7
35
0 
3 6 9 4
89
0 
5 4 6 9 1 5
20
0 
0 
87
0 
6 8 6 4 9 5 10
85
0 
3 5 4 6
89
0 
8 7 10 2 3 1 9 4 5
82
0 
9 6 7 8 9 2 5 1 10 4
0
0 
5 8 2 10 1 6
59
0 
1 10
71
0 
2 5 9
37
0 
7 10 1 7 3 2 5 9
66
0 
8 4 3 9 1 7 2 6 8
88
0 
0 
78
0 
1 3
14
0 
4 6 7 8 5
55
0 
8 9 3 1 8 7 10 2 6
21
0 
8 3 4 1 8 9 10 2 7
93
0 
7 5 6 4 1 9 10 8
82
0 
7 2 1 3 9 4 6 7
16
0 
8 8 5 10 1 7 9 2 3
55
0 
2 4 1
4
0 
9 7 6 2 3 1 4 8 5 10
34
0 
10 2 6 5 1 10 9 4 8 7 3
60
0 
10 8 4 9 7 3 1 10 2 6 5
2
0 
7 4 3 5 8 10 9 1
33
0 
8 7 10 5 6 9 4 3 1
98
0 
7 10 5 1 3 6 8 4
45
0 
4 8 2 7 1
21
0 
4 10 6 5 7
45
0 
4 9 2 3 7
84
0 
8 7 1 6 10 5 2 9 4
50
0 
5 1 2 7 6 5
47
0 
2 4 9
67
0 
4 1 7 3 4
30
0 
5 1 5 9 2 7
67
0 
8 3 1 2 4 10 8 7 6
94
0 
7 1 6 9 2 3 5 10
23
0 
10 2 8 5 7 4 3 9 10 1 6
6
0 
8 3 4 10 2 7 9 6 1
20
0 
6 4 2 3 1 8 5
87
0 
10 5 6 9 2 7 10 1 4 3 8
89
0 
10 2 3 8 7 4 1 9 5 6 10
98
0 
7 6 5 9 7 4 3 10
51
0 
6 3 4 6 8 7 2
0
0 
9 6 1 4 10 2 7 9 8 3
70
0 
8 9 10 1 4 3 5 2 8
29
0 
10 2 5 6 9 1 10 3 8 7 4
45
0 
1 5
58
0 
0 
60
0 
3 6 1 4
20
0 
7 3 5 9 8 2 4 10
28
0 
1 7
68
0 
4 6 3 8 5
10
0 
10 2 8 6 10 5 7 4 3 1 9
90
0 
1 6
0
0 
1 10
64
0 
5 4 9 5 10 7
50
0 
5 4 7 2 3 9
54
0 
3 7 2 5
38
0 
10 7 3 8 9 1 5 6 10 4 2
5
0 
9 10 8 6 9 7 5 2 3 1
15
0 
8 7 3 9 1 8 4 10 6
1
0 
2 3 1
89
0 
0 
74
0 
9 8 9 2 3 10 1 7 5 6
11
0 
7 10 3 9 6 4 2 8
44
0 
3 9 8 1
97
0 
9 10 3 1 2 8 5 9 4 6
68
0 
1 9
70
0 
7 5 3 7 1 2 8 9
52
0 
6 2 7 10 9 1 5
11
0 
4 9 10 8 1
69
0 
5 4 8 5 6 10
23
0 
7 6 9 4 1 10 2 5
37
0 
3 10 6 3
86
0 
2 7 10
77
0 
5 1 10 7 2 3
50
0 
1 2
68
0 
7 3 8 1 7 9 2 10
13
0 
5 3 2 4 10 8
74
0 
7 2 1 7 10 4 3 9
93
0 
10 9 4 8 7 6 3 1 2 5 10
27
0 
4 9 8 1 6
38
0 
2 4 3
56
0 
6 3 1 10 2 8 7
64
0 
7 2 9 5 8 4 6 1
60
0 
4 5 7 3 10
87
0 
1 2
38
0 
10 2 3 8 4 6 9 1 5 7 10
70
0 
5 9 5 6 4 7
74
0 
10 5 7 10 2 4 8 6 9 3 1
34
0 
9 2 5 4 3 1 10 6 7 8
85
0 
10 10 8 6 4 5 7 9 3 2 1
46
0 
5 10 7 3 4 6
79
0 
3 4 1 6
33
0 
4 6 2 9 3
3
0 
5 7 6 4 10 1
81
0 
4 10 1 3 8
30
0 
2 5 1
47
0 
6 5 6 7 2 3 9
86
0 
6 8 5 7 3 10 6
5
0 
9 2 10 1 6 3 5 4 9 8
6
0 
7 2 1 6 7 5 10 3
19
0 
5 3 7 9 2 4
63
0 
8 2 6 9 4 10 5 8 3
96
0 
2 1 4
95
0 
3 1 8 6
20
0 
5 4 5 1 2 9
43
0 
1 9
93
0 
10 5 10 6 3 1 7 9 8 4 2
68
0 
0 
44
0 
3 9 3 2
36
0 
7 4 7 1 8 6 3 10
74
0 
9 8 5 6 2 3 1 9 7 4
28
0 
6 9 10 3 5 1 8
21
0 
6 3 5 8 4 6 1
70
0 
1 8
14
0 
1 4
35
0 
9 3 6 8 9 1 4 5 10 7
41
0 
2 10 8
64
0 
10 2 7 1 6 4 9 3 8 10 5
24
0 
6 2 7 4 3 10 8
28
0 
9 3 1 6 4 5 9 2 8 10
3
0 
7 2 6 7 3 4 1 5
3
0 
7 9 7 1 5 2 3 6
93
0 
10 5 3 8 6 2 7 1 4 10 9
58
0 
1 4
73
0 
7 3 9 7 5 2 1 4
38
0 
4 7 9 10 3
84
0 
2 9 1
23
0 
5 4 8 3 6 10
56
0 
0 
5
0 
6 6 1 5 3 4 7
1
0 
5 3 4 7 1 2
86
0 
0 
37
0 
2 9 10
55
0 
6 4 2 8 1 6 5
24
0 
10 5 3 10 8 7 9 1 6 4 2
76
0 
8 6 8 9 10 5 3 2 4
76
0 
6 1 2 10 9 5 6
99
0 
7 4 2 6 8 3 10 7
57
0 
9 2 5 9 6 1 8 7 10 4
22
0 
7 4 3 10 7 9 1 2
63
0 
8 10 5 9 7 1 4 6 2
5
0 
1 4
69
0 
5 3 9 4 10 8
16
0 
4 2 8 5 4
38
0 
8 1 3 8 10 2 7 6 9
63
0 
5 1 8 5 10 6
69
0 
1 5
21
0 
9 5 9 10 8 3 1 4 2 7
16
0 
8 6 10 5 3 7 2 8 1
34
0 
1 2
33
0 
0 
51
0 
7 1 2 6 7 3 8 9
57
0 
0 
43
0 
10 1 7 10 6 2 9 5 8 3 4
60
0 
4 8 1 10 4
83
0 
9 2 9 6 1 8 3 4 7 5
74
0 
10 10 5 4 8 6 9 7 2 1 3
98
0 
0 
98
0 
2 8 6
23
0 
0 
15
0 
0 
23
0 
9 10 3 9 6 7 4 8 1 2
5
0 
10 9 4 1 2 3 10 6 7 5 8
45
0 
10 6 7 9 10 5 4 2 1 8 3
100
0 
2 3 7
39
0 
10 9 1 10 6 8 5 4 2 7 3
54
0 
5 1 4 6 8 7
51
0 
7 9 1 10 7 6 4 2
40
0 
2 1 7
51
0 
3 10 5 2
2
0 
9 8 9 10 1 4 3 7 5 6
38
0 
7 6 3 1 7 4 5 10
28
0 
10 2 3 6 9 10 8 7 1 5 4
79
0 
7 3 10 4 2 1 6 9
18
0 
7 4 3 2 9 8 10 1
97
0 
2 3 4
39
0 
5 5 3 9 10 2
40
0 
10 7 2 3 1 6 8 9 10 4 5
92
0 
6 6 10 9 7 3 2
98
0 
4 5 8 2 10
27
0 
4 8 10 3 2
99
0 
6 10 9 3 5 8 4
7
0 
0 
42
0 
4 5 2 1 9
72
0 
8 4 3 7 6 1 9 8 5
38
0 
3 10 6 2
89
0 
8 8 4 9 1 7 3 2 10
18
0 
1 1
55
0 
0 
89
0 
6 6 9 7 3 8 2
89
0 
1 9
10
0 
8 6 2 5 4 9 8 1 10
7
0 
0 
47
0 
5 6 1 8 4 10
71
0 
2 8 4
78
0 
5 8 5 3 7 6
17
0 
2 4 2
53
0 
2 4 1
78
0 
9 6 7 5 4 3 2 1 10 8
95
0 
1 9
43
0 
8 8 10 4 2 9 5 3 7
49
0 
0 
95
0 
9 9 10 2 4 8 7 1 6 5
25
0 
8 6 2 1 9 8 3 4 7
97
0 
3 6 1 5
83
0 
10 4 5 8 2 3 10 6 9 1 7
42
0 
8 7 2 3 9 6 1 5 10
63
0 
1 4
5
0 
1 9
67
0 
9 7 3 8 1 5 4 10 9 2
31
0 
2 10 6
74
0 
1 1
9
0 
1 10
67
0 
7 10 3 6 1 5 2 7
75
0 
1 10
23
0 
10 9 10 1 2 5 4 8 3 7 6
40
0 
10 6 2 5 10 7 8 1 3 9 4
55
0 
8 3 10 7 5 2 6 8 4
80
0 
1 4
73
0 
4 4 5 6 2
82
0 
7 8 1 9 4 7 5 6
66
0 
3 4 3 2
27
0 
10 2 10 1 4 6 3 8 7 9 5
99
0 
3 5 8 1
45
0 
10 10 7 5 8 6 1 4 2 3 9
100
0 
1 3
37
0 
2 4 9
13
0 
8 8 2 1 10 5 7 9 4
29
0 
2 4 1
92
0 
0 
28
0 
1 7
39
0 
9 10 5 9 6 4 1 8 3 2
4
0 
5 5 2 3 6 7
5
0 
10 5 9 7 8 6 3 10 1 4 2
26
0 
8 7 8 3 9 5 6 2 10
89
0 
1 2
22
0 
7 3 5 7 8 10 1 9
72
0 
4 8 2 3 1
84
0 
3 3 5 9
52
0 
7 9 4 2 3 6 7 1
90
0 
2 2 3
68
0 
9 4 5 3 6 2 9 8 1 7
77
0 
2 8 9
55
0 
2 3 9
21
0 
9 7 6 5 3 10 9 2 8 1
38
0 
3 1 4 2
0
0 
8 9 4 6 7 5 3 2 1
8
0 
6 2 5 1 8 6 10
5
0 
2 9 10
52
0 
10 5 8 2 3 10 6 1 7 4 9
18
0 
5 4 9 6 1 5
18
0 
2 7 6
22
0 
10 9 10 2 7 8 3 4 1 5 6
84
0 
10 1 3 2 8 5 6 7 9 10 4
94
0 
2 1 3
33
0 
9 1 2 10 9 8 3 4 7 6
76
0 
9 5 4 7 6 8 9 3 2 1
54
0 
10 8 5 6 1 9 10 3 4 2 7
44
0 
1 7
81
0 
3 4 8 1
80
0 
8 5 3 1 2 6 7 10 8
83
0 
6 2 3 6 1 7 5
10
0 
10 5 10 3 4 1 7 2 9 8 6
35
0 
7 3 1 8 4 10 9 2
44
0 
5 10 6 9 4 1
27
0 
8 1 6 7 9 3 8 4 10
49
0 
5 10 5 3 1 6
65
0 
3 6 4 10
37
0 
7 10 5 7 6 2 8 9
5
0 
7 9 6 10 8 3 5 4
3
0 
10 10 5 9 8 1 3 4 2 7 6
13
0 
3 5 10 1
37
0 
0 
44
0 
3 5 3 9
85
0 
7 1 3 2 4 6 8 9
53
0 
2 4 8
71
0 
10 2 3 10 7 9 4 1 6 5 8
28
0 
6 3 7 4 8 5 2
100
0 
4 2 9 7 4
57
0 
5 4 5 9 3 7
73
0 
7 3 10 1 4 5 7 8
37
0 
6 9 3 4 7 1 8
65
0 
7 9 2 4 10 3 8 7
64
0 
8 9 8 1 5 3 7 2 6
73
0 
4 8 6 5 1
24
0 
9 6 3 4 1 7 10 9 8 2
92
0 
3 7 3 2
80
0 
9 2 3 1 7 8 4 9 10 6
56
0 
9 9 7 4 5 1 2 3 6 8
11
0 
3 1 7 9
38
0 
8 10 9 1 6 8 2 5 4
80
0 
5 4 1 9 7 10
6
0 
2 10 4
18
0 
7 10 6 4 9 2 7 8
39
0 
4 7 4 2 9
7
0 
9 5 1 7 2 6 8 10 3 9
97
0 
1 1
98
0 
9 10 3 5 2 6 4 8 1 9
55
0 
9 1 5 10 2 6 4 7 3 9
11
0 
7 6 1 3 5 9 7 2
29
0 
6 2 8 1 6 9 10
75
0 
7 3 6 10 5 8 2 7
12
0 
8 6 4 5 1 10 8 2 9
99
0 
7 2 3 6 7 8 10 5
75
0 
4 5 10 4 9
11
0 
2 2 5
13
0 
4 2 3 8 5
68
0 
10 3 6 8 7 9 10 1 5 2 4
7
0 
7 2 4 6 5 1 9 10
25
0 
2 3 7
50
0 
8 2 1 4 3 7 9 10 8
86
0 
9 9 2 8 7 3 4 10 1 6
71
0 
0 
15
0 
8 1 10 8 7 2 5 4 9
47
0 
3 5 10 7
71
0 
8 8 6 5 4 3 7 9 1
33
0 
1 4
86
0 
4 2 4 5 3
23
0 
8 3 6 7 8 2 4 5 10
65
0 
5 1 3 4 5 2
17
0 
3 9 3 2
62
0 
4 1 3 6 2
31
0 
7 9 3 1 7 5 2 4
6
0 
6 9 5 1 2 3 6
63
0 
8 3 10 4 1 7 8 9 2
73
0 
10 10 8 2 9 1 4 7 3 5 6
47
0 
7 3 5 7 8 6 4 2
40
0 
9 6 2 7 3 5 1 9 8 10
78
0 
6 2 3 5 4 10 9
91
0 
5 2 9 5 4 8
23
0 
2 10 4
66
0 
1 3
52
0 
1 7
57
0 
1 9
96
0 
5 9 10 5 4 7
32
0 
6 6 3 4 10 7 9
63
0 
8 4 3 1 8 6 5 7 2
46
0 
0 
50
0 
4 1 10 3 4
86
0 
8 5 9 2 3 7 4 10 6
24
0 
1 5
39
0 
4 3 2 7 5
79
0 
0 
37
0 
10 7 4 3 6 10 8 5 2 9 1
33
0 
7 2 6 5 1 9 10 8
39
0 
8 8 9 2 5 10 1 3 6
72
0 
7 1 7 10 2 3 5 8
19
0 
5 8 2 7 9 5
26
0 
8 4 6 2 9 5 3 7 10
7
0 
4 6 8 7 3
87
0 
2 1 7
86
0 
9 5 8 6 9 10 7 1 2 4
27
0 
10 8 7 9 2 5 3 10 1 4 6
56
0 
5 3 5 7 4 10
42
0 
1 7
13
0 
7 10 8 7 9 6 2 4
78
0 
0 
54
0 
5 4 8 7 5 1
9
0 
1 6
70
0 
7 4 2 6 3 1 7 8
17
0 
0 
93
0 
4 1 2 5 10
14
0 
8 4 1 3 7 2 6 8 5
8
0 
8 7 3 2 6 5 4 1 10
58
0 
5 3 4 2 8 7
44
0 
0 
0
0 
6 4 1 8 5 3 9
20
0 
5 2 5 6 9 4
28
0 
3 7 1 10
65
0 
7 1 7 8 3 5 6 10
37
0 
3 10 3 9
37
