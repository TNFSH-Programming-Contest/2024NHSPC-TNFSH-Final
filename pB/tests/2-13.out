37
110237973
15
175
222
40
85
1839018
8
1217761847
81
1217761895
59
50
129600735
56
51
977928236
7
1838942
237988142
1845886
44
92
6743
131434923
40
129595790
1217761814
96
16
87
180
78
108392324
5186
1839028
131439833
60
87
74
205
129595778
1088166080
1744
108392623
44
1217759988
77
237994741
5
37
110238091
51
43
15
1217761873
108392348
6736
129595761
42
89
47
1838962
95
47
115
28
1728
108394026
135
1217756985
100
44
1840743
129597485
169
239833644
15
128
131441466
76
50
37
131441495
32
54
0
131434745
